monai/inferers/inferer.py

```diff
@@ -122,6 +122,9 @@ class SlidingWindowInferer(Inferer):
             `inputs` and `roi_size`. Output is on the `device`.
         progress: whether to print a tqdm progress bar.
         cache_roi_weight_map: whether to precompute the ROI weight map.
+        cpu_thresh: when provided, dynamically switch to stitching on cpu (to save gpu memory)
+                    when input image volume is larger than this threshold (in pixels/volxels).
```

Comment:
The indent of doc-string is not correct here.

Thanks.

Reply:
I've updated this doc-string